controllers/post_reply.rb

```diff
@@ -0,0 +1,12 @@
+class Chittter < Sinatra::Base
+
+  post '/reply' do
+    reply = Reply.create(reply: params[:reply],
+                         user: session[:username])
+    peep = Peep.first(id: session[:reply_id])
+    peep.replys << reply
+    peep.save
```

Comment:
if we would have proper associations we could do something like this:
peep.replies.create(....)